spark/dl/src/main/scala/com/intel/analytics/bigdl/models/lenet/Test.scala

```diff
@@ -50,8 +56,12 @@ object Test {
       val transformer =
         BytesToGreyImg(28, 28) -> GreyImgNormalizer(testMean, testStd) -> GreyImgToSample()
       val evaluationSet = transformer(rddData)
-
-      val model = Module.load[Float](param.model)
+      val url = new java.io.File(param.folder + "/model.bigdl").toURI.toURL()
```

Comment:
You should enhance the original FileReader to make it support http/https instead of modifying example one by one

Reply:
This a demo used to show the method works, I have enhanced File and FileReader in their own class.

Reply:
Then why change this file???